### ISSUE TITLE
Analysis of deterministic affine dynamics

### ISSUE THREAD

**Issue by schillic** [JuliaReach/ReachabilityAnalysis.jl#378]
Is this really not supported?
```julia
julia> b = [1.0]
1-element Array{Float64,1}:
 1.0

julia> prob = @ivp(x' = x + b, x(0) ∈ Singleton([1.0]))
InitialValueProblem{LinearControlContinuousSystem{Float64,IdentityMultiple{Float64},IdentityMultiple{Float64}},Singleton{Float64,Array{Float64,1}}}(LinearControlContinuousSystem{Float64,IdentityMultiple{Float64},IdentityMultiple{Float64}}([1.0], [1.0]), Singleton{Float64,Array{Float64,1}}([1.0]))

julia> sol = solve(prob, T=4.0)
ERROR: ArgumentError: the system type LinearControlContinuousSystem{Float64,IdentityMultiple{Float64},IdentityMultiple{Float64}} is currently not supported
```

---

Side note (probably an issue with `MathematicalSystems`):
```julia
julia> prob = @ivp(x' = -x + b, x(0) ∈ Singleton([1.0]))
ERROR: LoadError: Base.Meta.ParseError("\"*\" is not a unary operator")
Stacktrace:
 [1] @ivp(::LineNumberNode, ::Module, ::Vararg{Any,N} where N) at .julia/dev/MathematicalSystems/src/macros.jl:970
in expression starting at REPL[19]:1
caused by [exception 1]
Base.Meta.ParseError("\"*\" is not a unary operator")
```

**Comment by mforets** [comment]
> Side note (probably an issue with MathematicalSystems):

yes, i think this one is reported https://github.com/JuliaReach/MathematicalSystems.jl/issues/194

**Comment by mforets** [comment]
i take back https://github.com/JuliaReach/MathematicalSystems.jl/issues/221 , that normalization does not apply (because U is the universal set, so discretization fails). on the other hand i think that the proper fix is to make the `@ivp` macro be more precise, as proposed in https://github.com/JuliaReach/MathematicalSystems.jl/issues/221

**Comment by mforets** [comment]
this is an alternative for the OP:

```julia
julia> A = hcat(1.0); b = [1.0];

julia> prob = @ivp(AffineContinuousSystem(A, b), x(0) ∈ [1.0]);

julia> sol = solve(prob, T=4.0);
```

**Comment by schillic** [comment]
So it is not a problem of this package, sorry. From my side this can be closed.

**Comment by mforets** [comment]
The OP is now fixed on the MathematicalSystems.jl side, and we can do

```julia
prob = @ivp(x' = x + [1.0], x(0) ∈ Singleton([1.0]))
sol = solve(prob, T=4.0)
plot(sol, vars=(0, 1))
```
![Screenshot from 2020-12-11 17-41-23](https://user-images.githubusercontent.com/12424594/101952872-1cf93180-3bd8-11eb-8186-8be33260590c.png)

i'll add this example as a test before closing the issue.
